### PR TITLE
Update CWP information

### PR DESCRIPTION
**Pull request by jouvin**
An attempt to update/augment the information on CWP on our web site...

### CODE REVIEW

**Comment by jouvin** [comment]
Comments?

File: workinggroups/_posts/2016-07-06-communitywhitepaper.md [hunk 1]
```diff
@@ -4,10 +4,14 @@ layout: plain
 ---
 
 We have started the process for producing a Community White Paper (CWP) on software and computing to present the overall strategy and roadmap. 
-This activity is organised under the umbrella of the HSF.
+This activity is organised under the umbrella of the HSF. After the preparatory phase during Fall 2016, the CWP process will be launched in January 2017 at UCSD (January 23-27). Several working groups covering various aspects of the computing challenges ahead of us will work together to produce a final document at the end of 2017.
+
+To join the CWP process and get information about its progress, join [hsf-community-white-paper](https://groups.google.com/forum/#!forum/hsf-community-white-paper) Google group (no Google account is required, you can register by sending an email to [hsf-community-white-paper+subscribe@googlegroups.com](mailto:hsf-community-white-paper+subscribe@googlegroups.com)). Subscribing to this group with a Google account is required if you want to be able to contribute to CWP documents hosted in Google Docs.
```
Comment (hegner) [inline workinggroups/_posts/2016-07-06-communitywhitepaper.md]: join *the* .... group

File: workinggroups/_posts/2016-07-06-communitywhitepaper.md [hunk 2]
```diff
@@ -4,10 +4,14 @@ layout: plain
...
 
 # Resources
 
-  - [Charge from WLCG project](/assets/CWP-Charge-HSF.pdf)
+- [CWP presentation @CHEP2016](https://indico.cern.ch/event/505613/contributions/2323238/attachments/1352966/2043354/20161011-chep-cwp-plenary.pdf)
+- [WG descriptions](http://bit.ly/2dcZZqa)
```
Comment (hegner) [inline workinggroups/_posts/2016-07-06-communitywhitepaper.md]: WG --> Working Group

**Comment by hegner** [comment]
I think that's a decent start. Do we want to add some more information about what those challenges are somewhere on the site and link to that?

**Pull request merged**